packages/compass-indexes/src/stores/store.spec.ts

```diff
@@ -35,15 +35,11 @@ describe('IndexesStore [Store]', function () {
         localAppRegistry: localAppRegistry,
         instance: fakeInstance as any,
         dataService: {
-          indexes: (
-            ns: string,
-            options: unknown,
-            callback: (...args: any[]) => void
-          ) => {
-            callback('err', []);
+          indexes: () => {
+            return Promise.resolve([]);
           },
           isConnected: () => true,
-        } as IndexesDataService,
+        } as unknown as IndexesDataService,
```

Comment:
What I don't get is why typescript didn't seem to complain about unknown before? Or maybe it did and we didn't notice because it is a test file.

Reply:
Yeah, weird, should've been caught before if it's a real issue, typecheck should validate test files too